docs/en_us/platform_api/source/overview.rst

```diff
@@ -17,14 +17,4 @@ Supported edX Platform API Modules
 The following edX Platform APIs are currently supported.
 
 * :ref:`edX Platform Enrollment API`
```

Comment:
I removed the deprecated APIs from the list of supported APIs here. Maybe the draft page is cached somewhere.